smallrye-reactive-messaging-http/src/main/java/io/smallrye/reactive/messaging/http/converters/CloudEventSerializer.java

```diff
@@ -0,0 +1,22 @@
+package io.smallrye.reactive.messaging.http.converters;
+
+import io.cloudevents.json.Json;
+import io.smallrye.reactive.messaging.cloudevents.CloudEventMessage;
+import io.vertx.reactivex.core.buffer.Buffer;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+public class CloudEventSerializer extends Serializer<CloudEventMessage> {
+
+ @Override
+  public CompletionStage<Buffer> convert(CloudEventMessage payload) {
+    return CompletableFuture.completedFuture(Buffer.buffer(Json.encode(payload)));
```

Comment:
this uses CloudEvents Json class, which adds `Jdk8Module` and some (De)Serializers for `ZonedDateTime` - all with Jackson.